content/docs/data/Zapier.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Zapier'
+meta_title: 'Appbase - Zapier'
+meta_description: 'Automate your appbase.io database operations with Zapier.'
+keywords:
+    - dataschema
+    - appbase
+    - zapier
+    - elasticsearch
+sidebar: 'docs'
+---
+
+Connect the apps you use everyday to automate your work and be more productive. 1500+ apps and easy integrations - get started in minutes.
+
+You can read more about the zapier [here](https://zapier.com).
+
+## Overview
+
+I use a variety of tools to insert a record into appbase.io database such as https://importer.appbase.io. The issue is that these tools are fragmented and new records can come from a variety of sources such as Typeform, Google Sheets, etc.
```

Comment:
Since this is a doc reference, we can phrase it as an official statement. Remove the use of "I". You can frame it from a user's perspective (i.e. "You").

Additionally, we should phrase it differently. Example phrasing:

You can import data into appbase.io using any of the following methods:

1. Import via the dashboard GUI,
2. Import via ABC CLI,
3. Use the Zapier integration to import from over 1,500+ supported apps such as Typeform, Google Sheets, Mailchimp.

In this doc, we will walkthrough the process of importing using this Zapier integration using Google Sheets as an example input source.